USER_MANUAL.md

```diff
@@ -1539,6 +1550,58 @@ The follow-up [delay](#delay-follow_up) parameter can be set to require multiple
 
 **Notes of caution:** Follow-up thresholds are explored in detail in Fox et al. 2021\. Choosing follow-up rules is complex and work practices should be developed following extensive analysis of different scenarios. It is important to understand how follow-up thresholds and follow-up ratios interact, especially if both are to be used in the same program. Note that follow-up thresholds are similar to minimum detection limits but that the former is checked against to the measured emission rate (which is a function of quantification error) while the latter is checked against the true emission rate.
 
+### rolling_average
```

Comment:
Think this is missing the stationary parameter tag

Reply:
I didn't put it on this one as it was the "high-level" category that doesn't actually require user inputs. This is being consistent with the way I did the propagating parameter tags.